client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -502,6 +502,47 @@ const labelStatusTask = ( orderId, siteId, labelId, retryCount ) => {
 		} );
 };
 
+const handlePrintFinished = ( orderId, siteId, dispatch, getState, hasError, labels ) => {
+	dispatch( exitPrintingFlow( orderId, siteId, true ) );
+	dispatch( clearAvailableRates( orderId, siteId ) );
+
+	if ( hasError ) {
+		return;
+	}
+
+	if ( shouldEmailDetails( getState(), orderId, siteId ) ) {
+		const trackingNumbers = labels.map( ( label ) => label.tracking );
+		const carrierId = first( labels ).carrier_id;
+		const carrierName = 'usps' === carrierId ? translate( 'USPS' ) : translate( 'an unknown carrier' );
```

Comment:
It would probably be more future-proof to, if carrierName is unknown, simply say "Your order consisting of %(packageNum)d package has been shipped" below instead of having this odd string.